CONTRIBUTING.md

```diff
@@ -403,6 +403,7 @@ Environment variables can be used to modify the behaviour of VSCode. The followi
 -   `AWS_TOOLKIT_AUTOMATION`: If tests are currently being ran
 -   `DEVELOPMENT_PATH`: The path to the aws toolkit vscode project
 -   `AWS_TOOLKIT_TEST_NO_COLOR`: If the tests should include colour in their output
+-   `E2E_TESTING_DIRECTORY` - Where the e2e tests should find the tests to run. If not specified use `src/testE2E`
```

Comment:
We will be doing some refactoring of the structure so this may become outdated.

```suggestion
-   `E2E_TESTING_DIRECTORY` - Where the e2e tests should find the tests to run. If not specified it uses the default.
```